### DISCUSSION TITLE
How do I "save" my settings?

### DISCUSSION THREAD

**Discussion by harisund-yahoo** [valinet/ExplorerPatcher#2171] [Q&A]
I have access to multiple Windows 11 machines. I have some settings on my primarily laptop, and I want to replicate the settings across all the other servers I have access to. 

is there anyway I can do that? Like an .ini file I can copy from my laptop and put it somewhere on the other machines so the same settings are preserved?

**Comment by pyrates999** [comment]
Open up EP properties application.  Go to About on left side.  You should see import and export settings options.

**Reply by Poopooracoocoo** [reply]
I think they want to automate importing it?

**Reply by pyrates999** [reply]
Then a command line option would be the answer.  But I would do the following checks first before importing:

1. Check the EP version it was exported from
2. Check the Windows version, 10 or 11, and version number with build, such as 22621.2283, it was exported from
3. If the EP version doesn't match, don't allow to import it
4. If the Windows version, and version number with build doesn't match, don't allow to import it.

**Reply by harisund-yahoo** [reply]
Thank you, I am getting to what @pyrates999 is saying pretty much. 

I don't want a _hard hammer_ that says "don't allow importing" all together. I would like to be told that there might be some incompatibilities etc

**Reply by pyrates999** [reply]
Without checking every single option, it's easier to just only allow it if all the checks for the versions match.  Otherwise, don't allow it.
